src/strict_fstream.hpp

```diff
@@ -1,5 +1,4 @@
-#ifndef __STRICT_FSTREAM_HPP
-#define __STRICT_FSTREAM_HPP
+#pragma once
```

Comment:
should we revert to the more standard header guards, as non-standard pragma might potentially fail? https://en.wikipedia.org/wiki/Pragma_once
Being that the project is small, the overhead of the ifdefs is manageable?.

Reply:
I used to use #define's everywhere, but there's no compilers not supporting `#pragma once` anymore and I got stung by typos in the #define's a couple of times. :-P

and __FOO is reserved for the libc anyways, IIRC, so if we want to go back we have to switch to something a bit more likely to conflict as well, but it's trivial to do with guard2once (it can convert both ways)

Reply:
sounds good, let's leave it with pragma as you suggest.